Packs/VirusTotal/Integrations/VirusTotalV3/VirusTotalV3.py

```diff
@@ -2297,26 +2298,31 @@ def file_sandbox_report_command(client: Client, args: dict) -> CommandResults:
     assert isinstance(limit, int)  # mypy fix
     raise_if_hash_not_valid(file_hash)
     raw_response = client.file_sandbox_report(file_hash, limit)
-    data = raw_response['data']
-    return CommandResults(
-        f'{INTEGRATION_ENTRY_CONTEXT}.SandboxReport',
-        'id',
-        readable_output=tableToMarkdown(
-            f'Sandbox Reports for file hash: {file_hash}',
-            [
-                {
-                    'id': item['id'],
-                    **item['attributes'],
-                    'link': item['links']['self']
-                } for item in data
-            ],
-            headers=['analysis_date', 'last_modification_date', 'sandbox_name', 'link'],
-            removeNull=True,
-            headerTransform=underscoreToCamelCase
-        ),
-        outputs=data,
-        raw_response=raw_response
-    )
+    if 'data' in raw_response:
+        data = raw_response['data']
+        return CommandResults(
+            f'{INTEGRATION_ENTRY_CONTEXT}.SandboxReport',
+            'id',
+            readable_output=tableToMarkdown(
+                f'Sandbox Reports for file hash: {file_hash}',
+                [
+                    {
+                        'id': item['id'],
+                        **item['attributes'],
+                        'link': item['links']['self']
+                    } for item in data
+                ],
+                headers=['analysis_date', 'last_modification_date', 'sandbox_name', 'link'],
+                removeNull=True,
+                headerTransform=underscoreToCamelCase
+            ),
+            outputs=data,
+            raw_response=raw_response
+        )
+    elif raw_response.get('error', {}).get('code') == 'NotFoundError':
+        return CommandResults(readable_output=f'{file_hash} not found.')
+    else:
+        raise DemistoException("Error API Quota Exceeded.")
```

Comment:
Hello Martin, thanks a lot for your contribution. Have you consider to return a `CommandResults` when quota exceeded error? that is the usual behavior in the rest of the commands like file_command: https://github.com/VirusTotal/content/blob/master/Packs/VirusTotal/Integrations/VirusTotalV3/VirusTotalV3.py#L1811

Reply:
Hi @danipv,
thanks for your feedback. Great point. I adjusted the returned CommandResults to align to the usual behavior in the integration code. Please review the last commit.

Reply:
@danipv i also leveraged the same build_quota_exceeded_file_output and build_unknown_file_output functions as i think the same output applies to the sandbox reports as well.

Reply:
Love that. Thanks a lot. @jbabazadeh LGTM.